tests/product_price_printing/test.js

```diff
@@ -0,0 +1,24 @@
+var file_path = process.env.FILE_PATH
+const webdriver = require("selenium-webdriver");
+const assert = require("chai").assert
+const chromedriver = require('chromedriver')
+const { Builder, By } = require("selenium-webdriver");
```

Comment:
You are not using `assert`, `chromedriver` and `By` in this test. You should remove it.